src/Symfony/Component/Ldap/Adapter/ExtLdap/Connection.php

```diff
@@ -145,7 +145,7 @@ private function connect()
     private function disconnect()
     {
         if ($this->connection && is_resource($this->connection)) {
-            ldap_close($this->connection);
+            ldap_unbind($this->connection);
```

Comment:
This is the more confusing change ... but if you look up this function: http://php.net/ldap_close you see that `ldap_close` is just an alias of `ldap_unbind`.

Reply:
Indeed. Some aliases are also not documented, for my inspection lots of aliases had to be extracted from the source code :)